app-text/poppler/poppler23-23.09.0.recipe

```diff
@@ -271,41 +271,13 @@ PROVIDES_devel="
 	poppler23${secondaryArchSuffix}_devel = $portVersion
 	devel:libpoppler$secondaryArchSuffix = $libVersionCompat
 	devel:libpoppler_cpp$secondaryArchSuffix = $libCppVersionCompat
-	devel:libpoppler_glib$secondaryArchSuffix = $libGlibVersionCompat
-	devel:libpoppler_qt5$secondaryArchSuffix = $libQt5VersionCompat
-	devel:libpoppler_qt6$secondaryArchSuffix = $libQt6VersionCompat
 	"
 REQUIRES_devel="
-	haiku${secondaryArchSuffix}_devel
 	poppler23$secondaryArchSuffix == $portVersion base
-	lib:libpoppler_glib$secondaryArchSuffix
-	lib:libpoppler_qt5$secondaryArchSuffix
-	lib:libpoppler_qt6$secondaryArchSuffix
+	# poppler-glib
 	devel:libcairo$secondaryArchSuffix
-	devel:libcurl$secondaryArchSuffix
-	devel:libfontconfig$secondaryArchSuffix
-	devel:libfreetype$secondaryArchSuffix
-	devel:libgio_2.0$secondaryArchSuffix
 	devel:libglib_2.0$secondaryArchSuffix
 	devel:libgobject_2.0$secondaryArchSuffix
-	devel:libiconv$secondaryArchSuffix
-	devel:libintl$secondaryArchSuffix
-	devel:libjpeg$secondaryArchSuffix
-	devel:liblcms2$secondaryArchSuffix
-#	devel:libnspr4$secondaryArchSuffix
-#	devel:libnss3$secondaryArchSuffix
-	devel:libopenjp2$secondaryArchSuffix
-	devel:libpng16$secondaryArchSuffix
-	devel:libQt5Core$secondaryArchSuffix
-	devel:libQt5Gui$secondaryArchSuffix
-	devel:libQt5Widgets$secondaryArchSuffix
-	devel:libQt5Xml$secondaryArchSuffix
-	devel:libQt6Core$secondaryArchSuffix
-	devel:libQt6Gui$secondaryArchSuffix
-	devel:libQt6Widgets$secondaryArchSuffix
-	devel:libQt6Xml$secondaryArchSuffix
-	devel:libtiff$secondaryArchSuffix
-	devel:libz$secondaryArchSuffix
```

Comment:
These were introduced with the first poppler recipe but shouldn't actually be needed (as seen with pkg-config checks and looking into the *.pc files when installed)
First recipe: https://github.com/haikuports/haikuports/pull/984/files

Reply:
The only ones kept in for now are needed for poppler-glib-devel, adding them into that package wouldn't make sense to have in a base package.